Bio/SCOP/__init__.py

```diff
@@ -498,7 +497,7 @@ def write_des_sql(self, handle):
 
 
 class Node(object):
-    """ A node in the Scop hierarchy
+    """A node in the Scop hierarchy
```

Comment:
Since you'll have to edit this file again anyway, you might as well add the trailing full stop here too.